src/styles/blocks/card.scss

```diff
@@ -0,0 +1,130 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  background-color: $background-color;
+  max-width: 200px;
+  transition: transform 300ms;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $font-color;
+    margin: 40px 16px 0;
+  }
+
+  &__product-code {
+    font-weight: 400;
```

Comment:
it seems like you're using `font-weight` a lot, please consider using variable for that